node/src/server-modules/GlideJson.ts

```diff
@@ -100,8 +105,8 @@ export class GlideJson {
      * const result = await GlideJson.set("doc", "$", jsonStr);
      * console.log(result); // 'OK' - Indicates successful setting of the value at path '$' in the key stored at `doc`.
      *
-     * const jsonGetStr = await GlideJson.get(client, "doc", "$"); // Returns the value at path '$' in the JSON document stored at `doc` as JSON string.
-     * console.log(jsonGetStr); // '[{"a":1.0,"b":2}]'
+     * const jsonGetStr = await GlideJson.get(client, "doc", {path: "$"}); // Returns the value at path '$' in the JSON document stored at `doc` as JSON string.
+     * console.log(jsonGetStr); // '[{"a":1.0, "b":2}]'
```

Comment:
```suggestion
     * console.log(jsonGetStr); // '[{"a":1.0,"b":2}]'
```
actually it returns strings without spaces by default